packages/app/src/runs/RunsContainer.vue

```diff
@@ -190,6 +204,22 @@ const props = defineProps<{
 const showConnectSuccessAlert = ref(false)
 const connectionFailed = computed(() => !props.gql.currentProject?.cloudProject && props.online)
 
+const noRunsForBranchMessage = computed(() => {
+  const learnMoreLink = getUrlWithParams({
+    url: 'https://on.cypress.io/git-info',
+    params: {
+      utm_source: getUtmSource(),
```

Comment:
/nit: I think `getUrlWithParams` automatically applies a `utm_source` value which ends up overwriting any you supply